setup.py

```diff
@@ -37,7 +37,7 @@
         "sqlalchemy",
         "flask_login",
         "flask",
-        "pandas==1.0.5",
+        "pandas",
```

Comment:
This "change" also reverts a date precision bug fix that causes 3.6 unit tests to fail due to altered `pandas` behavior since its most recent update. This change needs to be removed.